src/containers/trends/TrendsSVGContainer.js

```diff
@@ -67,9 +68,27 @@ export class TrendsSVGContainer extends React.Component {
     ]);
   }
 
+  renderNoDataMessage(dataType) {
+    const { containerHeight: height, containerWidth: width, margins } = this.props;
+    const xPos = (width / 2) - margins.left + margins.right;
+    const yPos = (height / 2) - margins.top + margins.bottom;
+    const messagePosition = { x: xPos, y: yPos };
+    return (
+      <NoData
+        dataType={dataType}
+        position={messagePosition}
+      />
+    );
+  }
+
   renderCbg() {
-    const { containerHeight: height, containerWidth: width } = this.props;
     if (this.props.showingCbg) {
+      if (_.isEmpty(this.props.cbgData)) {
+        return this.renderNoDataMessage();
```

Comment:
And related: I think it should be pretty easy to set up some tests with the right props, so that we can *test* that we're rendering the correct no data message for cbg vs. smbg. Since this is a current 🐛 in the code, I think this would be a good idea.